Source/WebCore/platform/graphics/skia/SkiaPaintingEngine.cpp

```diff
@@ -65,6 +65,8 @@ SkiaPaintingEngine::SkiaPaintingEngine(unsigned numberOfCPUThreads, unsigned num
         m_cpuWorkerPool = WorkerPool::create("SkiaCPUWorker"_s, numberOfCPUThreads);
 }
 
+SkiaPaintingEngine::~SkiaPaintingEngine() = default;
```

Comment:
Since you've moved this from the header, I imagine you got a build error due to forward declaration and missing types. Is it because of unified builds that I don't see the same problem locally, or were you experimenting and this is a left-over?

Reply:
Yes, this is because of unified builds, it usually happens when a commit adds or removes files from compilation, the compilation unit changes and existing build failures appear, like this one.